perception/navigator_vision/nodes/cvod/cvod_utils/label_map_util.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Seems like these utils are included in tensorflow which will be installed as a dependency, right?